magprime/__init__.py

```diff
@@ -16,6 +16,14 @@ def SEASON_BADGE_PRICE(self):
     def SEASON_EVENTS(self):
         return magprime_config['season_events']
 
+    @property
```

Comment:
Clumsy, but we need to override this property to let people select a new badge type.